log-insights/setup/opentelemetry.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Log Insights: Kubernetes with OpenTelemetry'
+backlink_href: /docs/log-insights/setup
+backlink_title: 'Log Insights: Setup'
+---
+
+If you are running the database in the Kubernetes cluster, you can use the
+[`db_log_otel_server`](/docs/collector/settings#self-managed-servers) setting to
+have the collector listen for OpenTelemetry log data model messages.
+When this is specified, the collector will run the HTTP server behind the scenes
+on the specified port to receive logs at the `/v1/logs` endpoint.
+This should be a local address like `0.0.0.0:4318` and it's recommended to use
+an unprivileged port number (1024 and up) to avoid running the HTTP server as
+root.
+
+**Note:** This is currently only supporting CloudNativePG logs, with the jsonlog
+format (available with Postgres 15 or above).
```

Comment:
```suggestion
**Note:** This currently only supports CloudNativePG logs, with the jsonlog
format (available with Postgres 15 or above).
```

Separately, should we explicitly reference the [log_destination](https://www.postgresql.org/docs/current/runtime-config-logging.html#GUC-LOG-DESTINATION) setting here? And maybe link it? Or is that not helpful in this context?

Reply:
Actually, after the research, turned out that this is not correct. Well, "This currently only supports CloudNativePG logs" is correct, but CloudNativePG actually uses `log_destination = 'csvlog'` (https://cloudnative-pg.io/documentation/1.23/postgresql_conf/) then they convert to JSON style log: https://cloudnative-pg.io/documentation/1.23/logging/#postgresql-log

I'll update to drop the latter part. Thanks for leaving comment on this one, this actually has been confusing me for a while so I'm glad that now I know what exactly CloudNativePG does and what/how we support.
FYI @lfittl as we chatted about this yesterday a bit